static/app/routes.tsx

```diff
@@ -1470,6 +1470,30 @@ function buildRoutes() {
       </Route>
     </Fragment>
   );
+  const releaseThresholdRoutes = (
```

Comment:
🤷 
Maybe this doesn't need to be a separate const, but figured it'd help differentiate responsibility